modules/openapi-generator/src/main/resources/dart/libraries/dio/serialization/built_value/class_discriminator.mustache

```diff
@@ -0,0 +1,20 @@
+extension {{classname}}DiscriminatorExt on {{classname}} {
+    String? discriminatorValue() {
+        {{#mappedModels}}
+        if (this is {{modelName}}) {
+            return r'{{mappingName}}';
+        }
+        {{/mappedModels}}
+        return null;
+    }
+}
+extension {{classname}}BuilderDiscriminatorExt on {{classname}}Builder {
+    String? discriminatorValue() {
```

Comment:
Maybe make these a getter, that way there is no need to call a function in the templates which is not always expected.